src/value.cpp

```diff
@@ -485,6 +485,14 @@ bool Value::is_a(Env *env, ValuePtr val) {
 }
 
 bool Value::respond_to(Env *env, ValuePtr name_val) {
+    if (respond_to_method(env, SymbolValue::intern("respond_to?")))
+        return send(env, SymbolValue::intern("respond_to?"), { name_val })->is_true();
```

Comment:
omg I totally forgot we had an override of send that takes an initializer list. 🤦‍♂️

brb, about to do a big find+replace... 😆